instances/service/CollaboratoryLab.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/service/CollaboratoryLab",
```

Comment:
Should reflect the name of the instance: "/EBRAINSCollaboratoryLab"

Reply:
fixed